csrc/python_frontend/fusion_definition.h

```diff
@@ -240,6 +250,17 @@ class NVF_API FusionDefinition : public FusionState {
 
  private:
   mutable std::optional<std::string> debug_output_ = std::nullopt;
+
+#ifdef NVFUSER_DISTRIBUTED
+  //! DO NOT COMMIT TO THESE CHANGES!
```

Comment:
Sounds like a scary comment. Did you intend to fix it?

Reply:
If you carry on with your multi device executor integration, it would just naturally go away after that :stuck_out_tongue_winking_eye:

Reply:
What does "DO NOT COMMIT TO THESE CHANGES!" mean? Are we not supposed to commit the following lines in this PR?

Reply:
I use too harsh a comment there. I'll rename it to `//! following code is to be refactored when multidevice executor is properly integrated`